app/views/layouts/application.html.erb

```diff
@@ -5,8 +5,8 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_pack_tag 'application', media: 'all' %>
-    <%= javascript_pack_tag 'application' %>
+    <%= stylesheet_link_tag "application" %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
```

Comment:
What's this change?

Reply:
As we’re removing webpacker, we can’t use the `*_pack_tag` helpers anymore.